### PR TITLE
Add `cross-origin-isolated` Permission Policy directive data

### PR DESCRIPTION
**Pull request by 0xedward**
#### Test results and supporting details

- https://chromestatus.com/feature/5690888397258752
- You can also confirm by looking at the top level frame's Permission Policy available features in devtools under the Application tab
- Created a test page and set the proper COOP and COEP, then checked if the crossOriginIsolated property was accessible

### CODE REVIEW

File: http/headers/Permissions-Policy.json [hunk 1]
```diff
@@ -318,6 +318,41 @@
...
+        "cross-origin-isolated": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/crossOriginIsolated",
+            "support": {
+              "chrome": {
+                "version_added": "87"
+              },
+              "chrome_android": {
+                "version_added": false
+              },
```
Comment (skyclouds2001) [inline http/headers/Permissions-Policy.json:330]: per https://chromestatus.com/feature/5690888397258752, support for android is also added

File: http/headers/Permissions-Policy.json [hunk 2]
```diff
@@ -318,6 +318,41 @@
             }
           }
         },
+        "cross-origin-isolated": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/crossOriginIsolated",
```
Comment (queengooborg) [inline http/headers/Permissions-Policy.json:323]: This MDN URL doesn't look quite right.  Let's remove the MDN URL until a proper page has been created for this header value.

**Comment by queengooborg** [comment]
Hi @0xedward!  Do you plan to return to this PR and make the requested changes (as well as fix merge conflicts)?

**Review by Elchi3 — APPROVED** [review]
Thanks @0xedward, I resolved the conflicts and addressed the review comments for you 👍

**Pull request merged**